mobile-widgets/qml/Settings.qml

```diff
@@ -635,6 +635,30 @@ TemplatePage {
 						rootItem.settingsChanged()
 					}
 				}
+				TemplateLabel {
+					text: qsTr("GFLow")
+				}
+				TemplateTextField {
+					id: gfLow
+					text: PrefTechnicalDetails.gflow
+					inputMask: "99"
+					onEditingFinished: {
+						PrefTechnicalDetails.gflow = gfLow.text
+						rootItem.settingsChanged()
+					}
+				}
+				TemplateLabel {
+					text: qsTr("GFHigh")
+				}
+				TemplateTextField {
+					id: gfHigh
+					text: PrefTechnicalDetails.gfhigh
+					inputMask: "99"
+					onEditingFinished: {
```

Comment:
One thing I remember from other encounters with TextFields... make sure that this signal always fires when you stop editing. I.e., if you just tap on a different page in the menu bar, does this signal fire? What if you use the main menu to switch to the dive list. Etc. I'm not saying that this is the wrong signal - I'm just remembering that this has caused us pain in the past.

Reply:
There are two signal handlers  available for the TextFields, "onAccepted" and "onEditingFinished", with the difference being that the Accepted signal is only emitted when <enter> or <return> is hit, while EditingFinished is also emitted when the text field loses focus. In both cases the signal is only supposed to be emitted if the text in the field meets the inputMask and any validators that have been included return true.